src/main.js

```diff
@@ -5,17 +5,32 @@ import { Accordion, initializeAccordion } from './components/Accordion/Accordion
 
 import { AddToDoField, initializeAddToDoField } from './components/AddToDoItem/AddToDoField';
 
+import { getTasksDataFromLocalStorage } from './helpers/storage';
+
+import { ToDoItem } from './components/ToDoItem/ToDoItem';
+
+
+
 // selectors
 const pageTitle = document.querySelector('#title');
 const accordion = document.querySelector('#accordion');
-const addToDoSection1 = document.querySelector('#todo-section');
+const addToDoSection = document.querySelector('#todo-section');
+
+// load data from local storage and render tasks
+const dataOnLoad = getTasksDataFromLocalStorage();
+export const renderTasks = () => {
+  dataOnLoad.forEach((task) => {
```

Comment:
Please keep naming convention the same in all of the places - if somethig here is named as "name" - inside ToDoItem it should have the same "name" property - it has "message" currently, unify it either in this or other way

Reply:
.